STATUS: INLINE SUCCESSFUL
MODIFIED: com.google.auth.appengine.AppEngineCredentialsTest.warnsDefaultCredentialsWithTransport
MODIFIED FILE: /home/lichengpeng/test/automated_script/googleapis/google-auth-library-java/appengine/javatests/com/google/auth/appengine/AppEngineCredentialsTest.java
CLEANER: com.google.auth.appengine.AppEngineCredentialsTest.warnsDefaultCredentials
CLEANER FILE: /home/lichengpeng/test/automated_script/googleapis/google-auth-library-java/appengine/javatests/com/google/auth/appengine/AppEngineCredentialsTest.java
POLLUTER: N/A
POLLUTER FILE: N/A
ORIGINAL CLEANER SIZE: 8
NEW CLEANER SIZE: 1
ELAPSED TIME: 186.6210000514984
==========================
@@ -288,0 +288,7 @@
+ {
+     try {
+         Credentials unused = AppEngineCredentials.getApplicationDefault();
+     } catch (IOException ex) {
+     // ignore - this may just fail for not being in a supported environment
+     }
+ }
